uavcan/node/port/433.GetStatistics.0.1.uavcan

```diff
@@ -1,30 +0,0 @@
-# This service is used to obtain statistical metrics of a port (either message or service).
```

Comment:
You said this had an error with "request/response handling" that led you to remove this. Can you elaborate please?

Reply:
Suppose there is a node that provides a service under some service-ID, and also has a client on the same service-ID. When you send a request with a given service-ID, you can't specify if you are referring to the client or the server.